### ISSUE TITLE
Move repostitory?

### ISSUE THREAD

**Issue by spficklin** [feltus/OSG-KINC#8]
Are there any objections to moving this repository over to our SystemsGenetics organization where the other KINC tools are at?

https://github.com/systemsgenetics

**Comment by feltus** [comment]
Sounds good to me


Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Stephen Ficklin <notifications@github.com> Date: 2/5/18  5:01 PM  (GMT-05:00) To: feltus/OSG-KINC <OSG-KINC@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [feltus/OSG-KINC] Move repostitory? (#8) 
Are there any objections to moving this repository over to our SystemsGenetics organization where the other KINC tools are at?
https://github.com/systemsgenetics

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/feltus/OSG-KINC","title":"feltus/OSG-KINC","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/feltus/OSG-KINC"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Move repostitory? (#8)"}],"action":{"name":"View Issue","url":"https://github.com/feltus/OSG-KINC/issues/8"}}}

**Comment by wpoehlm** [comment]
@spficklin @feltus My only concern with that is the fact that we have links to this repo in two papers (the osg-kinc paper and the oncotarget paper).  Looking at the insights tab, people are clicking on this link directly from the oncotarget paper.

**Comment by feltus** [comment]
Can we clone it and keep it both places?


Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: William Poehlman <notifications@github.com> Date: 2/5/18  6:46 PM  (GMT-05:00) To: feltus/OSG-KINC <OSG-KINC@noreply.github.com> Cc: Alex Feltus <alex.feltus@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [feltus/OSG-KINC] Move repostitory? (#8) 
@spficklin @feltus My only concern with that is the fact that we have links to this repo in two papers (the osg-kinc paper and the oncotarget paper).  Looking at the insights tab, people are clicking on this link directly from the oncotarget paper.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/feltus/OSG-KINC","title":"feltus/OSG-KINC","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/feltus/OSG-KINC"}},"updates":{"snippets":[{"icon":"PERSON","message":"@wpoehlm in #8: @spficklin @feltus My only concern with that is the fact that we have links to this repo in two papers (the osg-kinc paper and the oncotarget paper).  Looking at the insights tab, people are clicking on this link directly from the oncotarget paper.  "}],"action":{"name":"View Issue","url":"https://github.com/feltus/OSG-KINC/issues/8#issuecomment-363260249"}}}

**Comment by wpoehlm** [comment]
We could but I think that we should discuss how we are going to run KINC 2.0 when it is released.  Will that be a new workflow or will we just swap out the software?

**Comment by spficklin** [comment]
Yeah, good point.  So, you don't want to move this repository if you have links to it.  But we could create a copy of it on the SysGen organization and then edit the README of this repository to indicate that the repository has moved. This way folks coming from the paper link can easily see where to find the most recent version.

**Comment by spficklin** [comment]
@4ctrl-alt-del and I spoke about this.  We'll build in a way to make the new KINC run on OSG.  The challenge is the new file format, which is binary.  We will probably have to build in a merge function into KINC that will allow those files generated on OSG to be merged.
